app/uk/gov/hmrc/tariffclassificationfrontend/connector/AuthenticatedHttpClient.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.connector
+
+import akka.actor.ActorSystem
+import javax.inject.Inject
+import play.api.libs.json.Writes
+import play.api.libs.ws.WSClient
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
+import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
+import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+
+import scala.concurrent.Future
+
+class AuthenticatedHttpClient @Inject()(auditConnector: AuditConnector, wsClient: WSClient, actorSystem: ActorSystem)
+                                       (implicit val config: AppConfig)
```

Comment:
do you prefer `config` explicit or implicit?

Reply:
Doesnt matter, usually with constructors we have it explicit